nupic/algorithms/anomaly.py

```diff
@@ -140,6 +148,17 @@ def compute(self, activeColumns, predictedColumns,
     if self._movingAverage is not None:
       score = self._movingAverage.next(score)
 
+    # apply binary discretization if required
+    if self._binaryThreshold is not None:
```

Comment:
I am using `moving avg, then discretize` to have fewer but compact packets of anomaly detected events; now I don't see much utility in doing it the other way. 

Actualy, the problem of ordering when to do MA gives a lot of possibilities we could judge. 

`pure _ (opt. likelihood/weighted) _ (discretize) _` where `moving average` can be done at any `_`